### PR TITLE
Add blogpost for sprint 75

### PR DESCRIPTION
**Pull request by krauselukas**

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit 37180b70efaa786568d60043a99875712e3df32f

https://deploy-preview-251--openbuildservice.netlify.app

File: _posts/development/2020-04-21-notifications-round-two.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Previously we started to introduce a new notification area in the Open Build Service webui.
+The goal of this is to enable the users, to stay on one medium in order to get the work done.
+No need to continuously switch between the browser and their email-clients in order to see
+where an action is needed. This is about a full-fledged notification system in the UI that
+not just shows raw requests in a table.
+To reach our aim, we worked out new feature to bring all of this to the next level...
+
+## Notifications page improvements
+
+The notifications now have a bit of context of what they are and where they come from. Notitifications for comments now will display a fragment of that comment, notifications for new reviews have a fragment of the review text, new request notifications have a bit of the request description, and request changes have a bit of information about what changed to what.
```
Comment (hennevogel) [inline _posts/development/2020-04-21-notifications-round-two.md:16]: I know this is everything that is happening but it's a bit tedious to read. How about a screenshot showing the 4 different types?

File: _posts/development/2020-04-21-notifications-round-two.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Users can mark notifications done and undone again, and they'll be sent to the right comment when clicking on a notification for a new comment.
+
+## Notifications configuration overhaul
+
+We've added new ways of keep users in the loop of their requests. Users can now receive notifications via email, RSS feeds and in the OBS web page Notifications area.
```
Comment (hennevogel) [inline _posts/development/2020-04-21-notifications-round-two.md:22]:
```suggestion
We've added new ways of keeping you in the loop of your requests. You can now receive notifications via email, RSS feeds and on the notifications page.
```

File: _posts/development/2020-04-21-notifications-round-two.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+This is configurable, to do it, the Actions menu that appears when you are on the Notifications page has a link to the Notifications configuration page.
+
+There, for each event type, and for each role (Maintainer, Bug owneer and so on) the user can change how is getting those notifications. He or she can, for example get them through all available channels, or just only one, whatever works better.
```
Comment (hennevogel) [inline _posts/development/2020-04-21-notifications-round-two.md:26]:
```suggestion
You can configure under which conditions you receive notifications on the configuration page.
```

File: _posts/development/2020-04-21-notifications-round-two.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+title: Notifications round two!
+category: development
+---
+
+Previously we started to introduce a new notification area in the Open Build Service webui.
+The goal of this is to enable the users, to stay on one medium in order to get the work done.
+No need to continuously switch between the browser and their email-clients in order to see
+where an action is needed. This is about a full-fledged notification system in the UI that
+not just shows raw requests in a table.
+To reach our aim, we worked out new feature to bring all of this to the next level...
```
Comment (hennevogel) [inline _posts/development/2020-04-21-notifications-round-two.md:12]:
```suggestion
where an action is needed. In the end we want this to be a full-fledged notification system in the user interface that helps you to keep track of all the things that might require your action. Like answering the comments another packager has left on your project. Or keeping track of the state and reviews of the submit requests you've sent/received.
```

File: _posts/development/2020-04-21-notifications-round-two.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+There, for each event type, and for each role (Maintainer, Bug owneer and so on) the user can change how is getting those notifications. He or she can, for example get them through all available channels, or just only one, whatever works better.
+
+<img src="/images/posts/notifications_round_two/notifications_configuration.png" alt="Notifications configuration" width="500"/>
+
+## Paginate long lists of notifications
+
+You can't make an omelette without breaking eggs. Users that are involved in a lot of
+projects will receive a lot of notifications. Therefore we started to paginate
+the list of notifications. This prevents slow page loads and leads to a neatly arranged
+list.
```
Comment (hennevogel) [inline _posts/development/2020-04-21-notifications-round-two.md:35]:
```suggestion
the list of notifications.
```

File: _posts/development/2020-04-21-notifications-round-two.md (hunk 1 above)
Comment (DavidKang) [reply _posts/development/2020-04-21-notifications-round-two.md]: @hennevogel, I added some screenshot

File: _posts/development/2020-04-27-notifications-round-two.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+Now, you can configure under which conditions you receive each type of notification (email, RSS and web).
+This is can be done on the notifications configuration page.
+
+<img src="/images/posts/notifications_round_two/notifications_configuration.png" alt="Notifications configuration" width="600"/>
+
+
+## Notifications Filtered by Project
+
+When we are checking the notfications page, sometimes we are only interested on notifications related to a certain project.
```
Comment (danidoni) [inline _posts/development/2020-04-27-notifications-round-two.md:29]:
```suggestion
When we are checking the notifications page, sometimes we are only interested on notifications related to a certain project.
```

Comment (danidoni) [inline _posts/development/2020-04-27-notifications-round-two.md:29]:
```suggestion
When we are checking the notfications page, sometimes we are only interested in notifications related to a certain project.
```

Comment (DavidKang) [reply _posts/development/2020-04-27-notifications-round-two.md]: Done, thanks :smile_cat:

Comment (DavidKang) [reply _posts/development/2020-04-27-notifications-round-two.md]: Done, thanks :smile_cat:

**Comment by DavidKang** [comment]
@hennevogel, could you review it again? :smile_cat:

**Pull request merged**